app/Forms/Components/DateFilterForm.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace App\Forms\Components;
+
+use Filament\Forms;
+use Filament\Forms\Components\DateTimePicker;
+use Filament\Forms\Components\Select;
+use Filament\Forms\Form;
+
+class DateFilterForm
```

Comment:
Couple of things I noticed with the ranges if you have a `display_timezone` and select 24hrs, week or month the results past the UTC time drop off the charts.

I'm using `America/New_York` if you want to emulate that.

Reply:
Just to make sure we are talking about the same. 
You mean that when you select "last 24h" it shows you only the full day before (yesterday) and not the last 24h hours.

Like this, its now 26-9 07:30 so it should show the tests from 26-9 as well. 

![image](https://github.com/user-attachments/assets/23aeb5e1-5d72-4c12-982d-49dd73d0bb6b)

Reply:
Fix this by swithcing to `toDateTimeString`